src/main/java/ch/njol/skript/expressions/ExprPrefix.java

```diff
@@ -0,0 +1,66 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.util.common.AnyPrefixed;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Prefix")
+@Description({"The prefix of a team, a member of a team, a player, or anything that can have a prefix.",
+	"Please note that getting or changing a player's permission-plugin prefix requires Vault "
+		+ "and a compatible permissions plugin.",
+	"Getting or changing a team prefix is supported by default."
+})
+@Examples({
+	"on chat:",
+	"\tcancel event",
+	"\tbroadcast \"%player's prefix%%player's display name%: %message%\" to the player's world",
+	"",
+	"set the player's prefix to \"[&lt;red&gt;Admin&lt;reset&gt;] \"",
+	"",
+	"set {team}'s prefix to \"&lt;blue&gt;Blue Team&lt;reset&gt; \""
+})
+@Since("2.0")
+@RequiredPlugins({"Vault", "a chat plugin that supports Vault"})
```

Comment:
This should be specific for player prefixes

Reply:
I realised as I was making this that Vault actually supports group/whatever prefixes as well, but it's a little out of scope for this.